### DISCUSSION TITLE
PA ( Pressure Advance )

### DISCUSSION THREAD

**Discussion by PatrickDenis** [OrcaSlicer/OrcaSlicer#602] [General]
Question , if I have PA enable and configured in Klipper in my printer.cfg
do I need to enable it in the Slicer too ?

Does the Slicer has priority over Klipper ?

THank you in advance

**Comment by anmarcu** [comment]
What the slicer will do is similar with filament gcode SET_PRESSURE_ADVANCE ADVANCE=0.0X.
So basically if you have a PA value in your printer.cfg, at print start that value will be overwritten.
